cmd/run_test.go

```diff
@@ -0,0 +1,137 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/fs"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"strconv"
+	"testing"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/suite"
+)
+
+var testFileContentsTemplate = `---
+meta:
+  author: "go-ftw"
+  enabled: true
+  name: "mock-test.yaml"
+  description: "Test file for go-ftw"
+tests:
+  - # Standard GET request
+    test_title: 1234
+    stages:
+      - stage:
+          input:
+            dest_addr: "127.0.0.1"
+            method: "GET"
+            port: {{ .Port }}
```

Comment:
Of course! That's much better than using regular expressions to replace stuff... We should fix that in all of the test YAML's we use.

Reply:
I'm planning to overhaul all the testing stuff. This is part of the change ;)